app/services/releasecop_service.rb

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
+
 class ReleasecopService
   attr_accessor :project
 
-  LOG_LINE_EXPR = /^(?<sha>[0-9a-f]+) (?<date>[0-9\-]+) (?<message>.*) \((?<name>.*), (?<email>.*)\)\w*$/ # %h %ad %s (%an, %ae)
+  LOG_LINE_EXPR = /^(?<sha>[0-9a-f]+) (?<date>[0-9\-]+) (?<message>.*) \((?<name>.*), (?<email>.*)\)\w*$/.freeze
```

Comment:
We should keep the comment, or maybe just link to the git-log expression that it must match: https://github.com/joeyAghion/releasecop/blob/bc9c108f1c4ff33fefe75ffb0c44f5ac8822594d/lib/releasecop/comparison.rb#L5